lib/mllp/sender.ex

```diff
@@ -236,29 +234,23 @@ defmodule MLLP.Sender do
         message
       end
 
-    state.tcp.send(state.socket, payload)
-    |> case do
+    case state.tcp.send(state.socket, payload) do
       :ok ->
-        reply =
-          if Keyword.get(options, :expect_reply) do
-            receive_reply(state, Keyword.get(options, :reply_timout))
-          else
-            {:ok, :sent}
+        if Keyword.get(options, :expect_reply) do
```

Comment:
I don't like this form at all, but I would not opt to refactor in this PR. Let's do a "clean up" PR instead.